azure-iot-device/azure/iot/device/iothub/models/message.py

```diff
@@ -5,6 +5,7 @@
 # --------------------------------------------------------------------------
 """This module contains a class representing messages that are sent or received.
 """
+SECURITY_MESSAGE_INTERFACE_ID = "urn:azureiot:Security:SecurityAgent:1"
```

Comment:
>SECURITY_MESSAGE_INTERFACE_ID [](start = 0, length = 29)

is this a constant across everything ? Then should it be set in the `constant.py` file ?

Reply:
This is an interesting idea. I'm good with it in either location for the time being, but I could see the argument for both.

If this value will ever change, it should be in `constant.py` for sure though. Otherwise I'm fine with it here.